### ISSUE TITLE
Unreadable buttons in table drop confirmation dialog (linux / wine)

### ISSUE THREAD

**Issue by btry** [HeidiSQL/HeidiSQL#509]
## Steps to reproduce this issue

1. Open a database locally
2. Try to drop a table
3. See the unreadable buttons and hard to read table name (black over dark background

## Current behavior
![image](https://user-images.githubusercontent.com/14139801/51840999-d7962880-230d-11e9-8338-acf0242c7f0c.png)

## Expected behavior

Readable button labels in the dialog

## Environment

* **HeidiSQL version:** 
  10.1.0.5464 configured with dark material theme

* **Database system and version:** 
  mariadb  Ver 15.1 Distrib 10.3.12-MariaDB, for debian-linux-gnu (x86_64) using readline 5.2
* **Operating system:** 
Debian Sid / wine-4.0-rc6 (Debian 4.0~rc6-1) / xfce 4 (with a dark theme)

**Comment by btry** [comment]
Hi

No, this is not limited to this dialog. I observed an other dialog with the same issue when I filled my other issue.

**Comment by ansgarbecker** [comment]
Please attach screenshots of these, or at least name the dialogs here.
Although I'm nearly sure in that case this is not fixable.

**Comment by btry** [comment]
An example : adding an index without any column to a table

![image](https://user-images.githubusercontent.com/14139801/51846273-94da4d80-2319-11e9-880c-7bc0d2971807.png)

**Comment by CharlieEtienne** [comment]
@ansgarbecker let me know if it's related to Material Theme

**Comment by ansgarbecker** [comment]
@btry could you please try other themes, if the look is the same as in your above screeny?
If so, I guess "Windows" theme is ok?

**Comment by btry** [comment]
Hi

Yes, the windows theme works fine
![image](https://user-images.githubusercontent.com/14139801/51898700-c3a40280-23b1-11e9-86a0-8d412050c4c6.png)

**Comment by cookieguru** [comment]
Also seen with the sql import confirmation:

![screenshot of file import confirmation dialog](https://user-images.githubusercontent.com/1888809/113629877-dd89a180-961b-11eb-9b0d-a5f3a1510e91.png)

The progress dialog is readable but isn't themed:

![screenshot of file import progress dialog](https://user-images.githubusercontent.com/1888809/113630015-16c21180-961c-11eb-86b8-a8068ee0165d.png)


* Theme: material
* HeidiSQL version: 11.1.0.6116
* OS: Ubuntu 18.04
* Wine version: wine-3.0 (Ubuntu 3.0-1ubuntu1)

**Comment by cookieguru** [comment]
Also from the connection dialog:

![screenshot of confirm changes to session dialog](https://user-images.githubusercontent.com/1888809/113643862-29e2da80-9638-11eb-8f76-cb164987dc3f.png)

![screenshot of access denied message](https://user-images.githubusercontent.com/1888809/113643858-25b6bd00-9638-11eb-9eb2-852b7afc752d.png)

**Comment by btry** [comment]
Hi

I did some tests with latest version available a few months ago (Ubuntu 20.04), it seems there is no significant progress on this topic. Maybe this is quite hard to solve. I need dark theme to preserve my eyes, and the best workaround I found is to customize the colors of the "windows" environment through winecfg. I succeeded to have rather good results and less readability problems than using native themes of the HeidiSQL, if you don't care the Windows 95 style of decorations :).

**Comment by ansgarbecker** [comment]
There's a chance this is fixed in the latest build, which is compiled through the latest Delphi compiler and VCL libraries.

**Comment by ansgarbecker** [comment]
Yes, only the latest nightly build is yet built with the new compiler: 11.3.0.6315 (just use Help > Check for updates)

**Comment by btry** [comment]
Hi

The nightly build available today shows better results.

An example with the theme Windows10 SlateGray. I also tested Windows10 Dark
![image](https://user-images.githubusercontent.com/14139801/122024534-cfe55a80-cdc8-11eb-81aa-f359facf14b9.png)

![image](https://user-images.githubusercontent.com/14139801/122042807-e694ad00-cdda-11eb-8fdf-438cb7a21b97.png)


An other example 
![image](https://user-images.githubusercontent.com/14139801/122042229-2d35d780-cdda-11eb-9162-7e91e1984d7e.png)

**Comment by cookieguru** [comment]
Confirmed much better with Material 👍

**Comment by ansgarbecker** [comment]
Shout if still an issue with the latest release.

**Comment by btry** [comment]
Hi

I upgraded last week; I did not noticed any problem so far with Material.  Thank you !

**Comment by btry** [comment]
Hi,

I wish to reopen this issue. If I try to run a SQL query which produces an error, the modal dialog still has bad colors when it shows for a second time. The first is ok (depite it is not perfectly readable yet).

1st occurence
![image](https://user-images.githubusercontent.com/14139801/164216101-ef41f731-6868-48f7-856a-de369e0df34c.png)

2nd occurence, if I run the my SQL query again

![image](https://user-images.githubusercontent.com/14139801/164216166-698945b0-281d-4c2b-86b3-d53f7216f921.png)

As operating systems tells applications the type of color theme (dark or light), it may be important to specify that I use a dark theme.

**Comment by btry** [comment]
Yes, there is no issue with the "Windows" theme. 2 consecutive dialogs render fine 

![image](https://user-images.githubusercontent.com/14139801/164224033-dcf3f022-00d2-4e93-84ef-557dd73e70f1.png)

**Comment by ansgarbecker** [comment]
The native Linux version of HeidiSQL has no themes, but support for OS dark mode. I'd say the native Linux version of HeidiSQL should be preferred over Wine.
